src/rust/plugin-registry/src/static_files/hax_docker_analyzer.nomad

```diff
@@ -0,0 +1,284 @@
+variable "plugin_id" {
+  type        = string
+  description = "The ID for this plugin."
+}
+
+variable "tenant_id" {
+  type        = string
+  description = "The tenant's ID. Used in the plugin-execution-sidecar."
+}
+
+variable "plugin_artifact_url" {
+  type        = string
+  description = "The url that specifies which binary to run as the plugin."
+}
+
+variable "plugin_count" {
+  type        = number
+  default     = 1
+  description = "The number of instances of the plugin to run."
+}
+
+variable "aws_account_id" {
+  type        = string
+  description = "The account ID of the aws account that holds onto the plugin binaries."
+}
+
+variable "plugin_runtime_image" {
+  type        = string
+  description = "The container that will load and run the plugin"
+}
+
+variable "plugin_execution_sidecar_image" {
+  type        = string
+  description = "The container that will load and run the Generator Executor or Analyzer Executor"
+}
+
+variable "graph_query_proxy_image" {
+  description = "Container image for running Graph Query Proxy. Only specify this for Analyzers."
+  type        = string
+  default     = null
+}
+
+variable "rust_log" {
+  type        = string
+  description = "Controls the logging behavior of Rust-based services."
+}
+
+variable "observability_env_vars" {
+  type        = string
+  description = <<EOF
+With local-grapl, we have to inject env vars for Opentelemetry.
+In prod, this is currently disabled.
+EOF
+}
+
+job "grapl-plugin" {
+  datacenters = ["dc1"]
+  namespace   = "plugin-${var.plugin_id}"
+  type        = "service"
+
+  reschedule {
+    # Make this a one-shot job
+    attempts = 0
+  }
+
+  # We'll want to make sure we have the opposite constraint on other services
+  # This is set in the Nomad agent's `client` stanza:
+  # https://www.nomadproject.io/docs/configuration/client#meta
+  constraint {
+    attribute = "${meta.is_grapl_plugin_host}"
+    value     = true
+  }
+
+  group "plugin-execution-sidecar" {
+    count = var.plugin_count
+
+    network {
+      mode = "bridge"
+      // TODO i think? possibly cargo culted?
```

Comment:
This dns stanza provides access to Consul DNS. Right now this is only used to forward traces to the otel collector, but I'm working to remove the need for Consul DNS entirely and just use Nomad's service discovery

Reply:
Removing since that's what thomas says